GooglePlayInstant/Editor/PlayInstantQuickDeployWindow.cs

```diff
@@ -68,14 +69,29 @@ void OnGUI()
         private void OnGuiCreateBundleSelect() 
         {
             EditorGUILayout.LabelField("AssetBundle Creation", EditorStyles.boldLabel);
-            EditorGUILayout.LabelField("Use the Unity Asset Bundle Browser to select your game's main scene " + 
-                "and bundle it (and its dependencies) into an AssetBundle file.", EditorStyles.wordWrappedLabel);
+            EditorGUILayout.LabelField("Use the Unity AssetBundles Browser to select your game's main scene " + 
```

Comment:
Switch back to "Unity Asset Bundle Browser" since that's the name they use

Reply:
Resolved